approved_tags.yaml

```diff
@@ -58,6 +58,7 @@
 - { tag: "politica", description: "Politica" }
 - { tag: "pratiche", description: "Buone pratiche di programmazione" }
 - { tag: "privacy", description: "Privacy e gestione dei dati" }
+- { tag: "progettazione-software", description: "Architetture, pattern, strumenti, metodologie" }
```

Comment:
Lo stile è inconsistente: alcuni tag di due parole hanno il trattino in mezzo, altri nulla. A parte `erlang-elixir` che è un caso a parte, scegliamo uno stile e aderiamo a quello.